core/capabilities/compute/compute.go

```diff
@@ -213,12 +212,12 @@ func (c *Compute) createFetcher(workflowID, workflowExecutionID string) func(req
 		}, "/")
 
 		fields := req.Headers.GetFields()
-		headersReq := make(map[string]any, len(fields))
+		headersReq := make(map[string]string, len(fields))
 		for k, v := range fields {
-			headersReq[k] = v
+			headersReq[k] = v.String()
 		}
 
-		payloadBytes, err := json.Marshal(sdk.FetchRequest{
+		payloadBytes, err := json.Marshal(ghcapabilities.TargetRequestPayload{
```

Comment:
Nit on naming: TargetRequestPayload is a bit misleading -- Compute isn't a target it's an action.

Can we make this more generic, eg. webapi.Request/Response?